core/client/bower.json

```diff
@@ -5,12 +5,12 @@
     "codemirror": "5.2.0",
     "devicejs": "0.2.7",
     "ember": "1.13.10",
-    "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
-    "ember-cli-test-loader": "0.1.3",
-    "ember-data": "1.13.13",
+    "ember-cli-shims": "0.0.6",
+    "ember-cli-test-loader": "0.2.1",
+    "ember-data": "1.13.15",
```

Comment:
I think we should be making `ember-data` updates in a separate PR so that it's easier to track down any behaviour changes.